### ISSUE TITLE
Add coverage information

### ISSUE THREAD

**Issue by natir** [labsquare/fastQt#3]
Hi,

I work with 3th generation of sequencing, in this generation all read didn't have the same length.

So for Per Base Content graph (its an example) the end of graph become chaotic because the information is support just by a little bit of sequence.

![an example](https://pageshot.net/images/26768c94-4520-403a-b0cf-3b5fe6c6cf37.png)

May be we can add coverage information by color gradiant on background of the graph.

Thank for your work.

**Comment by dridk** [comment]
Good idea ! 
I may do it dynamically . For exemple, moving the cursor hover the line will display a popup with depth information.

**Comment by dridk** [comment]
I will make it dynamics by moving mouse cursor for the next release. Actually it's not pretty.

**Comment by natir** [comment]
Yes it's more pretty but when you export graph you can't keep this information. Maybe just add another x axis with this information is a good think. Comment the experimentale code for the 0.2 release.
